analyzers/dataframe/include/FCCAnalyses/Algorithms.h

```diff
@@ -92,6 +92,22 @@ namespace Algorithms{
     double _variable[3]={1.0,1.0,1.0};
   };
 
+  /// Calculates the thrust axis by looping over all possible combinations
+  struct calculate_thrust {
+    calculate_thrust(){}
+
+    inline void mag2(float (&vec)[4]);
+    inline float dot(float vec1[4], float vec2[4]);
+    inline void cross(float (&vec)[4], float vec1[4], float vec2[4]);
+    inline void unit(float (&vec)[4]);
+    inline void plus(float (&vec)[4], float vecIn1[4], float vecIn2[4]);
+    inline void minus(float (&vecOut)[4], float vecIn1[4], float vecIn2[4]);
+    inline void copy(float (&vecOut)[4], float vecIn[4]);
+
```

Comment:
I'm wondering if we should move those type of utilities to some Utils namespace as they have nothing specific to thrust calculation

Reply:
Moved those functions to `Utils.h`